.evergreen/config.yml

```diff
@@ -1640,6 +1594,7 @@ tasks:
           params:
             binary: bash
             working_dir: src
+            include_expansions_in_env: ["PYTHON_BINARY"]
```

Comment:
Is this a bug you caught during the fix?

Reply:
We need to pass this explicitly where needed now to use the correct Python binary. Part of our later work will be to unify the use of `PYTHON_BINARY` to remove the need to pass it explicitly.